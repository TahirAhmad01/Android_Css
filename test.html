<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body>
    <script>
      const path = require("path");
      const GoogleAssistant = require("google-assistant");
      const config = {
        auth: {
          keyFilePath: path.resolve(__dirname, "YOUR_API_KEY_FILE_PATH.json"),
          // where you want the tokens to be saved
          // will create the directory if not already there
          savedTokensPath: path.resolve(__dirname, "tokens.json"),
          // you can also pass an oauth2 client instead if you've handled
          // auth in a different workflow. This trumps the other params.
          oauth2Client: YOUR_CLIENT,
        },
        // this param is optional, but all options will be shown
        conversation: {
          audio: {
            encodingIn: "LINEAR16", // supported are LINEAR16 / FLAC (defaults to LINEAR16)
            sampleRateIn: 16000, // supported rates are between 16000-24000 (defaults to 16000)
            encodingOut: "LINEAR16", // supported are LINEAR16 / MP3 / OPUS_IN_OGG (defaults to LINEAR16)
            sampleRateOut: 24000, // supported are 16000 / 24000 (defaults to 24000)
          },
          lang: "en-US", // language code for input/output (defaults to en-US)
          deviceModelId: "xxxxxxxx", // use if you've gone through the Device Registration process
          deviceId: "xxxxxx", // use if you've gone through the Device Registration process
          deviceLocation: {
            coordinates: {
              // set the latitude and longitude of the device
              latitude: xxxxxx,
              longitude: xxxxx,
            },
          },
          textQuery: "What time is it?", // if this is set, audio input is ignored
          isNew: true, // set this to true if you want to force a new conversation and ignore the old state
          screen: {
            isOn: true, // set this to true if you want to output results to a screen
          },
        },
      };

      const assistant = new GoogleAssistant(config.auth);

      // starts a new conversation with the assistant
      const startConversation = (conversation) => {
        // setup the conversation and send data to it
        // for a full example, see `examples/mic-speaker.js`

        conversation
          .on("audio-data", (data) => {
            // do stuff with the audio data from the server
            // usually send it to some audio output / file
          })
          .on("end-of-utterance", () => {
            // do stuff when done speaking to the assistant
            // usually just stop your audio input
          })
          .on("transcription", (data) => {
            // do stuff with the words you are saying to the assistant
          })
          .on("response", (text) => {
            // do stuff with the text that the assistant said back
          })
          .on("volume-percent", (percent) => {
            // do stuff with a volume percent change (range from 1-100)
          })
          .on("device-action", (action) => {
            // if you've set this device up to handle actions, you'll get that here
          })
          .on("screen-data", (screen) => {
            // if the screen.isOn flag was set to true, you'll get the format and data of the output
          })
          .on("ended", (error, continueConversation) => {
            // once the conversation is ended, see if we need to follow up
            if (error) console.log("Conversation Ended Error:", error);
            else if (continueConversation) assistant.start();
            else console.log("Conversation Complete");
          })
          .on("data", (data) => {
            // raw data from the google assistant conversation
            // useful for debugging or if something is not covered above
          })
          .on("error", (error) => {
            // handle error messages
          });
      };

      // will start a conversation and wait for audio data
      // as soon as it's ready
      assistant
        .on("ready", () => assistant.start(config.conversation))
        .on("started", startConversation);

      //console.log(assistant);
    </script>
  </body>
</html>
