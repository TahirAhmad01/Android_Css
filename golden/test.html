<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<style>
* {
  box-sizing: border-box;
}

body {
  background-color: #474e5d;
  font-family: Helvetica, sans-serif;
}

/* The actual timeline (the vertical ruler) */
.timeline {
  position: relative;
  max-width: 1200px;
  margin: 0 auto;
}

/* The actual timeline (the vertical ruler) */
.timeline::after {
  content: '';
  position: absolute;
  width: 6px;
  background-color: white;
  top: 0;
  bottom: 0;
  left: 50%;
  margin-left: -3px;
}

/* Container around content */
.container {
  padding: 10px 40px;
  position: relative;
  background-color: inherit;
  width: 50%;
}

/* The circles on the timeline */
.container::after {
  content: '';
  position: absolute;
  width: 25px;
  height: 25px;
  
  background-color: white;
  border: 4px solid #FF9F55;
  top: 15px;
  border-radius: 50%;
  z-index: 1;
}

/* Place the container to the left */
.left {
  left: 0;
}

/* Place the container to the right */
.right {
  left: 50%;
}

/* Fix the circle for containers on the right side */
.right::after {
  left: -16px;
}

/* The actual content */
.content {
  padding: 20px 30px;
  background-color: white;
  position: relative;
  border-radius: 6px;
}

/* Media queries - Responsive timeline on screens less than 600px wide */
@media screen and (max-width: 600px) {
  /* Place the timelime to the left */
  .timeline::after {
  left: 31px;
  }
  
  /* Full-width containers */
  .container {
  width: 100%;
  padding-left: 70px;
  padding-right: 25px;
  }
  
  /* Make sure that all arrows are pointing leftwards */
  .container::before {
  left: 60px;
  border: medium solid white;
  border-width: 10px 10px 10px 0;
  border-color: transparent white transparent transparent;
  }

  /* Make sure all circles are at the same spot */
  .left::after, .right::after {
  left: 15px;
  }
  
  /* Make all right containers behave like the left ones */
  .right {
  left: 0%;
  }
}
</style>
</head>
<body>

<div class="timeline">
  <div class="container left">
    <div class="content">
      <h2>2017</h2>
      <p>Lorem ipsum dolor sit amet, quo ei simul congue exerci, ad nec admodum perfecto mnesarchum, vim ea mazim fierent detracto. Ea quis iuvaret expetendis his, te elit voluptua dignissim per, habeo iusto primis ea eam.</p>
    </div>
  </div>
  <div class="container right">
    <div class="content">
      <h2>2016</h2>
      <p>Lorem ipsum dolor sit amet, quo ei simul congue exerci, ad nec admodum perfecto mnesarchum, vim ea mazim fierent detracto. Ea quis iuvaret expetendis his, te elit voluptua dignissim per, habeo iusto primis ea eam.</p>
    </div>
  </div>
</div>

</body>
</html>
